File: AuraFit-BackEnd/src/main/java/app/aurafitbackend/DTOS/UserDTO.java
package app.aurafitbackend.DTOS;

import app.aurafitbackend.Enums.Role;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class UserDTO {
    private String firstName;
    private String lastName;
    private String email;
    private Role role;
}

File: aurafit-frontend/src/Models/DTOS/UserDTO.ts
import {Role} from "../Enums/Role.ts";

export class UserDTO {
    firstName: string
    lastName: string
    email: string
    role: Role

    constructor(firstName: string, lastName: string, email: string, role: Role) {
        this.firstName = firstName
        this.lastName = lastName
        this.email = email
        this.role = role
    }
}